rustls/src/crypto/hpke.rs

```diff
@@ -15,6 +15,7 @@ use crate::Error;
 ///
 /// [RFC 9180]: <https://www.rfc-editor.org/rfc/rfc9180.html>
 /// [HPKE ciphersuite profile]: <https://datatracker.ietf.org/doc/html/draft-ietf-tls-esni-17#section-9>
+#[doc(hidden)]
```

Comment:
AIUI this could be put on `pub mod hpke` in `crypto/mod.rs` to apply to the whole module

Reply:
I'll give that a shot, I wasn't sure if a top level attribute would work.

Reply:
That worked great, thanks for suggesting :+1: